src/widget/wtrackmenu.h

```diff
@@ -107,7 +107,7 @@ class WTrackMenu : public QMenu {
     void slotScaleBpm(mixxx::Beats::BpmScale scale);
 
     // Info and metadata
-    void slotUpdateReplaygainFromDeckGain();
+    void slotUpdateReplaygainFromPregain();
```

Comment:
"ReplayGain" vs. "Replaygain" is still inconsistent. Until now we used only "ReplayGain"

Reply:
done